k8s/clients.go

```diff
@@ -0,0 +1,93 @@
+package k8s
+
+import (
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/e2e-harness/pkg/harness"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	kubeConfigPath = harness.DefaultKubeConfig
```

Comment:
Can we make the kubeconfig path configurable rather than use the constant? As its a weird shipyard path?

`kind` will give us the path to the kubeconfig and we can wrap that in `e2ectl`.

```
$ kind get kubeconfig-path
/Users/ross/.kube/kind-config-kind
```

Reply:
Good shout. That would also work nice with multiple kind clusters in app-operator tests.

Does it make sense to default to "$HOME/.kube/kind-config-kind"?

Reply:
@rossf7 PTALA. I changed that so it's configurable. We can change the default when we start using kind so it will be enough to just vendor.

Reply:
Thanks. Good plan on changing the default when we switch over to kind.